File: 1036.py
from math import sqrt
A, B, C = input().split()
A = float(A)
B = float(B)
C = float(C)
if A == 0:
    print('Impossivel calcular')
if A > 0:
    delta = B**2-4*A*C
    if delta < 0:
        print('Impossivel calcular')
    elif delta > 0:
        quadra = sqrt(delta)
        x1 = (-B + quadra)/(2*A)
        if x1 == 0:
            print('Impossivel calcular')
        else:
           print(f'R1 = {x1:.5f}')
        x2 = (-B - quadra) / (2 * A)
        if x2 == 0:
            print('Impossivel calcular')
        else:
           print(f'R2 = {x2:.5f}')

File: 1036.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = lines.shift().split(" ");
var A =  Number(n[0]); var B =  Number(n[1]); var C =  Number(n[2]);

if (A == 0){
    console.log('Impossivel calcular')
}
if (A > 0){
    delta = B**2-4*A*C
    if (delta < 0){
        console.log('Impossivel calcular')
    }
    if (delta > 0){
        quadra = Math.sqrt(delta)
        x1 = (-B + quadra)/(2*A)
        if (x1 == 0){
            console.log('Impossivel calcular')
        }
        else{
           console.log(`R1 = ${(x1).toFixed(5)}`)
        }
        x2 = (-B - quadra) / (2 * A)
        if (x2 == 0){
            console.log('Impossivel calcular')}
        else{
            console.log(`R2 = ${(x2).toFixed(5)}`)}
    }
}
